python_modules/libraries/dagster-aws/dagster_aws_tests/aws_credential_test_utils.py

```diff
@@ -0,0 +1,34 @@
+import json
```

Comment:
what's the right place for this? Don't want to put it in _core/test_utils bc the boto dependency